Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM postgres:11.5
+FROM postgres:15.1
```

Comment:
I don't think we should upgrade posters, since a lot of application are stuck on 11.

Reply:
The issue is that the reason I upgraded is not that I needed version 15, but because the building the dockerfile in `dhl-parcel-db-docker` based on the older postgres version no longer works:

When `apt-get update` to install the extensions you get a 404, failing the Dockerfile build. With this change a new build can be made again. This means:
- If you want updated tables/configuration, postgres needs to be updated for it.
- If you don't, then there's no need to update your version of `dhl-parcel-db-docker` based on version 11.5, so just keep using that.
Apart from that, I imagine that even though you would want to have a version of postgres for tests that is the same version as in production, in practice things will probably still work the same.